deploy/modules/gcp/tidb-operator/versions.tf

```diff
@@ -0,0 +1,11 @@
+
+terraform {
+  required_version = ">= 0.12"
+  required_providers {
+    google      = "~> 2.16"
+    google-beta = "~> 2.16"
+    external    = "~> 1.2"
+    helm        = "~> 0.10"
+    null        = "~> 2.1"
+  }
+}
```

Comment:
`deploy/modules` do not contain version-lock file, so they will install the latest provider and may fail on `validate` command
xref: https://github.com/pingcap/tidb-operator/issues/1172